js/test/Common/controller.html

```diff
@@ -5,10 +5,6 @@
         <meta name="viewport" content="width=device-width, initial-scale=1.0" />
         <title>Test Suite | Ice for JavaScript</title>
         <link rel="icon" type="image/x-icon" href="/assets/favicon.ico" />
-        <link rel="stylesheet" type="text/css" href="/assets/common.css" />
```

Comment:
We can add a simple CSS later, but I didn't want to have dependencies in a big CSS framework like foundation.